static/styles/zulip.css

```diff
@@ -2114,10 +2114,6 @@ div.floating_recipient {
     float: none;
 }
 
-.streams_popover .popover_sub_unsub_button::after {
-    content: " Unsubscribe";
-}
-
```

Comment:
Yeah, that's a great idea. We should define in which cases using `content` is okay; as far as I know there are some cases when it's useful for CSS hacks (e.g. `content: " "`).